_health-care/_js/components/Nav.jsx

```diff
@@ -8,60 +8,102 @@ class Nav extends React.Component {
     return (
       <ul className="usa-sidenav-list">
         <li role="presentation">
-          <Link to="/introduction"
-              className={this.props.currentUrl === '/' || this.props.currentUrl === '/introduction' ? ' usa-current' : ''}>
+          <Link to="/introduction" activeClassName="usa-current">
```

Comment:
See react router API for link.

On Thu, Mar 10, 2016, 08:53 Anne Kainic <notifications@github.com> wrote:

> In _health-care/_js/components/Nav.jsx
> <https://github.com/department-of-veterans-affairs/vets-website/pull/1351#discussion_r55710410>
> :
>
> > @@ -8,60 +8,102 @@ class Nav extends React.Component {
> >      return (
> >        <ul className="usa-sidenav-list">
> >          <li role="presentation">
> > -          <Link to="/introduction"
> > -              className={this.props.currentUrl === '/' || this.props.currentUrl === '/introduction' ? ' usa-current' : ''}>
> > +          <Link to="/introduction" activeClassName="usa-current">
>
> Where does activeClassName come from?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/department-of-veterans-affairs/vets-website/pull/1351/files#r55710410>
> .
>